modules/apps/object/object-test-util/src/main/java/com/liferay/object/test/util/ObjectFieldLocalizedTestUtil.java

```diff
@@ -0,0 +1,129 @@
+import com.liferay.object.field.builder.TextObjectFieldBuilder;
+import com.liferay.object.field.setting.builder.ObjectFieldSettingBuilder;
+import com.liferay.object.model.ObjectField;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+import com.liferay.portal.kernel.util.ListUtil;
+import com.liferay.portal.vulcan.util.LocalizedMapUtil;
+
+import java.io.Serializable;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * @author Paulo Albuquerque
+ */
+public class ObjectFieldLocalizedTestUtil {
+
+	public static ObjectField addLocalizedObjectField(String businessType) {
+		return addLocalizedObjectField(businessType, 0);
+	}
+
+	public static ObjectField addLocalizedObjectField(
+		String businessType, long objectDefinitionId) {
+
+		if (Objects.equals(
+				businessType, ObjectFieldConstants.BUSINESS_TYPE_LONG_TEXT)) {
+
+			return new LongTextObjectFieldBuilder(
+			).labelMap(
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString())
+			).name(
+				"longTextLocalized"
+			).objectDefinitionId(
+				objectDefinitionId
+			).localized(
+				true
+			).build();
+		}
+
+		if (Objects.equals(
+				businessType, ObjectFieldConstants.BUSINESS_TYPE_RICH_TEXT)) {
+
+			return new RichTextObjectFieldBuilder(
+			).labelMap(
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString())
+			).name(
+				"richTextLocalized"
+			).objectDefinitionId(
+				objectDefinitionId
+			).localized(
+				true
+			).build();
+		}
+
+		if (Objects.equals(
+				businessType, ObjectFieldConstants.BUSINESS_TYPE_TEXT)) {
+
+			return new TextObjectFieldBuilder(
+			).labelMap(
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString())
+			).name(
+				"textLocalized"
+			).objectDefinitionId(
+				objectDefinitionId
+			).localized(
+				true
+			).objectFieldSettings(
+				Collections.singletonList(
+					new ObjectFieldSettingBuilder(
+					).name(
+						ObjectFieldSettingConstants.NAME_UNIQUE_VALUES
+					).value(
+						Boolean.TRUE.toString()
+					).build())
+			).build();
+		}
+
+		return null;
+	}
+
+	public static Map<String, Serializable> getLocalizedValues(
+		String... locales) {
+
+		List<String> localeList = ListUtil.fromArray(locales);
+
+		return HashMapBuilder.<String, Serializable>put(
+			"longTextLocalized_i18n",
+			HashMapBuilder.putAll(
+				_getValues(localeList)
+			).build()
+		).put(
+			"richTextLocalized_i18n",
+			HashMapBuilder.putAll(
+				_getValues(localeList)
+			).build()
+		).put(
+			"textLocalized_i18n",
+			HashMapBuilder.putAll(
+				_getValues(localeList)
+			).build()
+		).build();
+	}
+
+	private static Map<String, String> _getValues(List<String> locales) {
+		HashMap<String, String> values = new HashMap<>();
+
+		for (String locale : locales) {
+			values.put(locale, RandomTestUtil.randomString());
+		}
+
+		return values;
```

Comment:
@p-albuquerque I can't see the point of these two new methods. They're only used in two places and the code is no easier to understand than before.